content/source/docs/enterprise/sentinel/import/tfconfig.html.md

```diff
@@ -267,28 +267,14 @@ import "tfconfig"
 main = rule { tfconfig.module_paths contains ["foo"] }
 ```
 
-#### Iterating through modules
+#### Iterating Through Modules
 
 Iterating through all modules to find particular resources can be useful. This
-example shows how to use `module_paths` with the [`module()`
-function](#root-function-module) to retrieve all resources of a particular type from
-all modules (in this case, the [`azurerm_virtual_machine`][ref-tf-azurerm-vm]
-resource). Note the use of `else []` in case some modules don't have any
-resources; this is necessary to avoid the function returning undefined.
+[example](iterate-over-modules) shows how to use `module_paths` with the
+[`module()` function](#function-module-) to find all resources of a
+particular type from all modules using the `tfplan` import. By changing `tfplan` in this function to `tfconfig`, you could make a similar function find all resources of a specific type in the Terraform configuration.
```

Comment:
Please wrap this line to 79 characters to match the rest of the document.

Reply:
ok